tractor/__init__.py

```diff
@@ -99,16 +99,18 @@ def run(
     name: Optional[str] = None,
     arbiter_addr: Tuple[str, int] = (
         _default_arbiter_host, _default_arbiter_port),
-    # the `multiprocessing` start method:
+    # either the `multiprocessing` start method:
     # https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods
-    start_method: str = 'forkserver',
+    # OR `trio_run_in_process` (the new default).
+    start_method: Optional[str] = None,
```

Comment:
This should just default to the string for 'trip'. I think you set it to None here because that is what the `try_set_start_method` returns for trip. I don't think that's how that function should behave, so once that gets refactored this should just be a string type with whatever the string repr for that mode is.

Reply:
>understand the whole multiprocessing module load thing

If you're talking about the forkserver hack stuff, yeah don't worry about it. I'm starting to think we'll drop all that to write our own anyway.

Reply:
@salotz so as you can see from the build triggered after fb93336 which failed, on windows we need to have a different default. If `'trio_run_in_process'` is passed to `try_set_start_method()` we're going to get an error on windows (I think this is correct since we want that function to explicitly fail when an incorrect method is chosen on the platform).

This is the small dilemma, on different systems we require different default start methods. We can either leave it as I had it -> `None` is default and implies the correct actual default for the system is chosen internally *or* we move that same logic into `tractor.run()` so that's it's more clear that's what's happening?

Interested in your opinion here :)

Reply:
Yeah and the other reason to leave that default as `None` and *not* have the *default-for-the-system* logic here would be that when we get alternatives for windows then we'll want to pass through the user chosen value and validate it.